spec/Vehicle/Vehicle.vspec

```diff
@@ -191,6 +191,24 @@ AmbientAirTemperature:
   unit: celsius
   description: Ambient air temperature outside the vehicle.
 
+AmbientHumidity:
+  datatype: float
+  type: sensor
+  unit: percent
+  min: 0
+  max: 100
+  description: Relative humidity outside the vehicle.
+               0 = Dry, 100 = Air fully saturated.
+
+OutsideLightIntensity:
+  datatype: float
+  type: sensor
+  unit: percent
+  min: 0
+  max: 100
+  description: Light intensity outside the vehicle.
+               0 = No light detected, 100 = Fully lit.
```

Comment:
"Fully lit" is interesting as I'm not clear on how this can be assessed. (Relative to the sun's intensity?)

Reply:
That is the reason for the comment below. We see it as useful to have light intensity represented, as that can be used to control e.g. Daytime Running Lights and brightness of displays. A vehicle may e.g. decide to put on low beam if light intensity falls below 50%, or shift navigation to night mode on navigation if intensity falls below 30%, but how light intensity is calculated is likely vehicle dependent, i.e. there is as far as I know no de-facto industry standard saying that e.g. any Lux value above 20000 Lux shall be considered 100% and 1000 Lux be considered 50%. 

So in short, this leaves it to the OEM to decide on how to calculate brightness. This is similar to other signals like `Brake.IsBrakesWorn` where we also leave the logic and detailed meaning to the OEM.